# Copyright (c) 2024 Smart Rollerz e.V.
# All rights reserved.

pathplanning_node:
  ros__parameters:
    # subscriber
    lane_points_subscriber: '/lane_detection/result'
    image_subscriber: '/camera/birds_eye'
    state_machine_subscriber: '/state_machine/debug'
    state_machine_lane_subscriber: '/rightLane'
    remote_state_subscriber: '/remoteState'
    targetSteeringAngle_pub: '/targetSteeringAngle'


    # publisher
    image_debug_publisher: '/pathplanning/debug_image'
    path_planning_right_publisher: '/pathplanning/right_lane_coefficients'
    path_planning_left_publisher: '/pathplanning/left_lane_coefficients'
    ref_point_publisher: '/controller/ref_pose'         # maybe change to pathplanning

    # additional params
    trj_look_forward: 100      # adapt to speed and runtime
    active: True
